draft-ietf-rats-corim.md

```diff
@@ -487,16 +487,28 @@ A CoMID defines several types of Claims, using "triples" semantics.
 At a high level, a triple is a statement that links a subject to an object via a predicate.
 CoMID triples typically encode assertions made by the CoRIM author about Attesting or Target Environments and their security features, for example measurements, cryptographic key material, etc.
 
-The set of triples is extensible.
-The following triples are currently defined:
+This specification defines two sets of triples.
+The first set of triples is essential to the fundamental principles of Evidence appraisal as illustrated in {{-rats-arch}} and {{-rats-endorsement}} and is mandatory to implement (MTI).
+Every Verifier MUST support the MTI triples, if CoRIM processing is supported by the Verifier.
+The second set of triples is included in this specification as they are generally useful across profiles and are optional to implement (OTI).
+Ever Verifier SHOULD support OTI triples, if CoRIM processing is supported by the Verifier, unless there are good reasons not to do so, such as restrictions imposed by constrained resource environments, reduced code complexity in support of periodic code review, or scoping to Attester simplicity.
+
+MTI Triples:
 
 * Reference Values triples: containing Reference Values that are expected to match Evidence for a given Target Environment ({{sec-comid-triple-refval}}).
 * Endorsed Values triples: containing "Endorsed Values", i.e., features about an Environment that do not appear in Evidence. Specific examples include testing or certification data pertaining to a module ({{sec-comid-triple-endval}}).
+* Conditional Endorsement triples: describing one or more conditions that, once matched, result in augmenting the Attester's actual state with the supplied Endorsed Values ({{sec-comid-triple-cond-endors}}).
+* CoMID-CoSWID linking triples: associating a Target Environment with existing CoSWID Payload tags ({{sec-comid-triple-coswid}}).
```

Comment:
Why is coswid mandatory? CoSWID has its own tag type and can be processed as part of corim without impacting comid. There is significant work required to create a comid-to-coswid mapping. Processing of coswids could be done independent from processing of comids.

Reply:
There was some comment about it being in the base set of CoRIM contents: CoBOM, CoMID, CoSWID.

That's why I put it in there. The requirements might be external to the CoMID structure. The editors have to figure that out in the next meeting.

Reply:
> The requirements might be external to the CoMID structure.

Inclusion of different (from CoMID) tags in a corim doesn't obviate the need to map the other tags to CoMID. There can be a separate discussion (not necessarily in the CoRIM draft) for how to translate the external representation into an internal representation that doesn't have to be `intrep`. However, could be `intrep` too. That would be the topic for a design team to wrestle with. The coswid triples in comid could be lower priority to work on now since a mapping between a coswid internal representation with a comid internal representation would be useful pretext. It seems this sort of thing doesn't meet the criteria of something that is "base" capability for a comid engine.